src/main/java/net/sf/jabref/gui/PreviewPanel.java

```diff
@@ -306,9 +307,19 @@ public void update() {
         );
         String newValue = sb.toString();
 
-        previewPane.setText(newValue);
-        previewPane.revalidate();
-
+        if (SwingUtilities.isEventDispatchThread()) {
```

Comment:
Probably, yes. However, I'm not sure if that is a good idea in general. (Except for the fact that I do not really know how to pass it... As `Runnable`? Probably...